qemu/tests/qemu_guest_agent.py

```diff
@@ -894,6 +894,42 @@ def gagent_check_thaw_unfrozen(self, test, params, env):
             test.fail("The return value of thawing an unfrozen fs is %s,"
                       "it should be zero" % ret)
 
+    @error_context.context_aware
+    def gagent_check_query_chardev(self, test, params, env):
```

Comment:
gagent_query_chardev is ok.

Reply:
No, it is fixed format. 
chk_type = "gagent_check_%s" % gagent_check_type

Reply:
yes ,you are right.